docs/site/Deploying-to-IBM-Cloud.md

```diff
@@ -53,7 +53,7 @@ Cloudant connector, so data is persisted.
   and resource group. Select "Use both legacy credentials and IAM" as the
   available authentication methods
 
-  <img src="./imgs/deploytocloud-mycloudant.png" alt="myCloudant service name" style="width: 300px"/>
+  <img src="https://loopback.io/pages/en/lb4/imgs/deploytocloud-mycloudant.png" alt="myCloudant service name" style="width: 300px"/>
```

Comment:
I wonder why the relative path does not work. If we keep the same directory structure for loopback.io and github, it should work for both.

Reply:
FYI: on https://loopback.io/doc/en/lb4/Deploying-to-IBM-Cloud.html, the broken image is https://loopback.io/doc/en/lb4/imgs/deploytocloud-mycloudant.png.

Reply:
I remember having the same problem when adding a diagram in todo example (https://raw.githubusercontent.com/strongloop/loopback-next/master/examples/todo/README.md).  TV was looking at it and couldn't get it to work, and finally change the image url to `![todo-tutorial-overview](https://loopback.io/pages/en/lb4/imgs/todo-overview.png)`.

Reply:
Ideally, we should make it work for both github and loopback.io.